tutorials/flume-capture-events.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: page
+title: Capturing Events with Flume
+---
+
+## Purpose
+
+This lesson demonstrates how you can configure Flume to capture events from a web application with minimal impact on performance or the user. Flume collects individual events and writes them in groups to the dataset.
+
+The Flume agent receives the events over inter-process communication (IPC), and writes the events to the Hive file sink. Each time you send a message, Log4j writes a new `INFO` line in the terminal window.
+
+This example demonstrates how to generate Flume configuration information from the Kite CLI. In addition, JSP and servlet samples allow you to test the data capture mechanism.
+
+### Prerequisites
+
+* A VM or cluster configured with Flume user impersonation. See [Preparing the Virtual Machine][vm].
+* An [Events dataset][events] in which to capture session events.
+
+[vm]:{{site.baseurl}}/tutorials/preparing-the-vm.html
+[events]:{{site.baseurl}}/tutorials/create-events-dataset.html
+
+### Result
+
+Flume is configured to listen for events on a Tomcat server instance. Use the JSP and servlets to send events to Tomcat. Log4j logs each event to the terminal window. Flume stores the events in `dataset:hive:events`.
+
+## Configuring Flume
+
+Follow these steps to configure Flume to channel log information directly to the `events` dataset. You first generate the configuration information using the Kite command-line interface, copy the results, paste them in the Flume configuration file, and then restart Flume.
+
+You can configure Flume for this example using either Cloudera Manager or the command line.
+
+### Configuring Flume in Cloudera Manager
+
+1. In a terminal window, type `kite-dataset flume-config --channel-type memory events`.
```

Comment:
I think the command could use more explanation. What doe sit do and why? What might I change for production instead of the demo? (The channel would be a file channel in production, by the way)